.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -6,15 +6,15 @@ on:
     inputs:
       reference_url:
         description: The WebSocket url of the reference node
-        default: wss://rpc.polkadot.io
+        default: wss://kusama-rpc.polkadot.io
```

Comment:
Any reason we choose to default to kusama? Just curious :)

Reply:
There are 2 reasons :)
- to go from Kusama to Polkadot you just need to remove the Kusama part
- Kusama is usually what we check first for the releases